DataFormats/CTPPSAlignment/src/RPAlignmentCorrectionsData.cc

```diff
@@ -43,6 +43,8 @@ RPAlignmentCorrectionData& RPAlignmentCorrectionsData::GetSensorCorrection(unsig
   return sensors[id];
 }
 
+//----------------------------------------------------------------------------------------------------
+
 RPAlignmentCorrectionData RPAlignmentCorrectionsData::GetSensorCorrection(unsigned int id) const
 {
   RPAlignmentCorrectionData a;
```

Comment:
Using a single letter variable name is discouraged by the CMSSW code rules. I know that it was already as such also in the previous version of it, but I would profit of this opportunity to give it a more descriptive name.

The same in RPAlignmentCorrectionsData::GetRPCorrection